lighthouse-core/report/v2/report-template.html

```diff
@@ -27,11 +27,16 @@
 <body>
   <noscript>Lighthouse report requires JavaScript. Please enable.</noscript>
   <div hidden>%%LIGHTHOUSE_TEMPLATES%%</div>
+  <div id="lighthouse-content"></div>
+  <div id="lighthouse-log"></div>
   <script>%%LIGHTHOUSE_JAVASCRIPT%%</script>
   <script>window.__LIGHTHOUSE_JSON__ = %%LIGHTHOUSE_JSON%%;</script>
   <script>
-    const renderer = new ReportRenderer(document);
-    document.body.appendChild(renderer.renderReport(window.__LIGHTHOUSE_JSON__));
+    window.addEventListener('DOMContentLoaded', _ => {
+      window.renderer = new ReportRenderer(document);
+      const container = document.querySelector('#lighthouse-content');
```

Comment:
made sense to render into a container rather than replace the whole world for every call to render.